tests/python/pants_test/base/test_exception_sink_integration.py

```diff
@@ -209,12 +209,12 @@ def test_reset_interactive_output_stream(self):
 
     with temporary_dir() as tmpdir:
       some_file = os.path.join(tmpdir, 'some_file')
-      safe_file_dump(some_file, b'', mode='wb')
+      safe_file_dump(some_file, '')
```

Comment:
This is the only instance where I converted something that was previously bytes to be now unicode. It's safe to do here because we see the file is always created this same way (not parametrized) and just a few lines below we call `read_file` with Unicode.

The other usages were a bit less clear if it's safe, so I kept as is.